2-mandatory/4-people-i-know.js

```diff
@@ -378,13 +378,21 @@ First, I want you to find all of my friends who are 35 or older.
 */
 
 let thirtyFiveOrOlder = [];
+friends.forEach((item) => {
+  if (item.age >= 35) {
+    thirtyFiveOrOlder.push(item.name.first);
```

Comment:
Good work! Some complex stuff here but very well handled. One test fails (2 - friends that are over 35), it's related to this line. Any idea what?